themes/Delicious/foundation/typography.scss

```diff
@@ -43,6 +43,30 @@ $line-height-data: (
   display-small: (
     base: rem(24px),
     large-screen: rem(24px)
+  ),
+  heading-h1:(
+    base:rem(40px)
+  ),
+  heading-h2:(
+    base:rem(36px)
+  ),
+  heading-h3:(
+    base:rem(28px)
+  ),
+  heading-h4:(
+    base:rem(24px)
+  ),
+  heading-subheading:(
+    base:rem(24px)
+  ),
+  textbody-largetext:(
+    base:rem(24px)
+  ),
+textbody-medtext:(
```

Comment:
@Farhan-emgage fix indent. Also, use `text-body` for this map. There is not need to duplicate the "text" term. Simple use `text-body`,`text-body-large`, and `text-body-small`